pandas/tests/io/test_parquet.py

```diff
@@ -564,8 +565,19 @@ def test_s3_roundtrip(self, df_compat, s3_resource, pa, s3so):
             write_kwargs=s3so,
         )
 
-    @td.skip_if_no("s3fs")
-    @pytest.mark.parametrize("partition_col", [["A"], []])
+    @td.skip_if_no("s3fs")  # also requires flask
+    @pytest.mark.parametrize(
+        "partition_col",
+        [
+            pytest.param(
+                ["A"],
+                marks=pytest.mark.xfail(
+                    PY38, reason="Getting back empty DataFrame", raises=AssertionError,
```

Comment:
right is this something we are controlling on pandas side? or pyarrow / fastparquet?

Reply:
No idea cc @jorisvandenbossche ?

Reply:
ahh i c this is failing PRs ok let'e merge and can always come back to this (@jbrockmendel if you can make an issue about this)